lib/databricks.js

```diff
@@ -10,7 +20,65 @@ export class DatabricksSingleton {
     if (!DatabricksSingleton.instance) {
       try {
         DatabricksSingleton.instance = import("@databricks/sql").then(
-          (module) => module.default
+          (module) => {
+            const databricks = module.default;
+            const {TCLIService_types} = databricks.thrift;
+
+            const boolean = ["null", "boolean"],
+              integer = ["null", "integer"],
+              number = ["null", "number"],
+              object = ["null", "object"],
+              string = ["null", "string"];
+
+            DatabricksSingleton.types
```

Comment:
I have this question on https://github.com/observablehq/observablehq/pull/9761, too, and it may also be applicable here. I referenced https://docs.databricks.com/sql/language-manual/sql-ref-datatypes.html, but didn't handle every possible type. I'm wondering what would happen if a table contained, for instance, map or struct data?